src/main/webapp/css/user_style.css

```diff
@@ -0,0 +1,84 @@
+.main {
+    text-align: center;
+    padding: 0px 10px;
+    font-family: sans-serif;
+}
+
+.column {
+    display: inline-block;
+    width: 45%;
+}
+
+.autocomplete {
+    position: relative;
+    display: inline-block;
+}
+
+input {
+    border: 1px solid transparent;
+    background-color: #f1f1f1;
+    padding: 10px;
+    font-size: 16px;
+}
+
+input[type=text] {
+    background-color: #f1f1f1;
+    width: 100%;
+}
+
+input[type=submit] {
+    background-color: #4285F4;
+    color: #fff;
+}
+
+.autocomplete-items {
+    border: 1px solid #d4d4d4;
+    border-bottom: none;
+    border-top: none;
+    z-index: 99;
+    top: 100%;
+    left: 0;
+    right: 0;
+    max-height: 300px;
+    overflow: scroll;
```

Comment:
There's still some weird styling happening when the scroll bar disappears and there's just an awkward bottom and right border around the autocomplete items dropdown

I think the bottom border can be resolved by making this overflow-y: scroll. 

You might want to look into some of the `autocomplete-items::-webkit-scrollbar` styling, but I guess I'm just nitpicking at UI style at this point